### PR TITLE
fix markdown links, add markdown link checker

### PR DESCRIPTION
**Pull request by silesky**
#494

### CODE REVIEW

File: packages/browser/README.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ export const lowercase: Plugin = {
 }
 ```
 
-For further examples check out our [existing plugins](https://github.com/segmentio/analytics-next/tree/master/src/plugins).
+For further examples check out our [existing plugins](/packages/browser/src/plugins).
```
Comment (chrisradek) [inline packages/browser/README.md:226]: Is it possible to use the full url with the linter? I suspect these links will be broken on NPM if we change them to relative paths.

Comment (silesky) [reply packages/browser/README.md:226]: That's a good point -- `npm` _should_ resolve it automatically, but I just checked, and our repository field in the package.json looks wrong. My last commit should fix it. 

See: https://stackoverflow.com/questions/62575382/relative-link-from-readme-md-to-another-file-in-package-rendered-in-npmjs

Comment (silesky) [reply packages/browser/README.md:226]: I also added the repository field to the yarn constraints file, so its enforced.

**Review by pooyaj — APPROVED** [review]
❇️

**Pull request merged**